share/applications/kakoune.desktop

```
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Type=Application
+Version=
+Name=Kakoune
+GenericName=Text Editor
+Comment=Edit text files
+Icon=kakoune
+TryExec=kak
+Exec=kak %F
```

You should use `--` to prevent Kakoune from interpreting filenames as command-line options.
```suggestion
Exec=kak -- %F
```